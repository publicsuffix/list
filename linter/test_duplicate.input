// test:
// - valid wildcard usage
// - invalid wildcard usage

// ===BEGIN ICANN DOMAINS===

// *.com implicitely includes .com
com
*.com

// double exception
!www.com
!www.com

// double wildcard
*.example.com
*.example.com

// double plain rule
example1.com
example1.com

// redundant/overlapping rule
example.com

// ===END ICANN DOMAINS===
