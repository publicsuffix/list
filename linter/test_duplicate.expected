9: error: Found doublette/ambiguity (previous line was 8): '*.com'
13: error: Found doublette/ambiguity (previous line was 12): '!www.com'
17: error: Found doublette/ambiguity (previous line was 16): '*.example.com'
21: error: Found doublette/ambiguity (previous line was 20): 'example1.com'
24: error: Found doublette/ambiguity (previous line was 17): 'example.com'
26: warning: No PRIVATE section found
